### PR TITLE
[Tensorflow 2.7.4] Updating TF 2.7 wheels for version 2.7.4

### PR DESCRIPTION
**Pull request by dkey-amazon**
Updating the Tensorflow 2.7 point version from 2.7.0 to TF 2.7.4

*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Tests run
**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [X] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [X] `sagemaker_local_tests = true`

## Label Checklist
- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ohadkatz** [comment]
LGTM! 🚢

**Review by ohadkatz — DISMISSED** [review]

File: tensorflow/training/docker/2.7/py3/Dockerfile.ec2.cpu.os_scan_allowlist.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            "cvss_v3_score": 9.1,
+            "cvss_v30_score": 0.0,
+            "cvss_v31_score": 9.1,
+            "cvss_v2_score": 0.0,
+            "cvss_v3_severity": "CRITICAL",
+            "source_url": "https://snyk.io/vuln/SNYK-PYTHON-TENSORFLOW-1315149",
+            "source": "SNYK",
+            "severity": "CRITICAL",
+            "status": "ACTIVE",
+            "title": "SNYK-PYTHON-TENSORFLOW-1315149 - tensorflow"
```
Comment (shantanutrip) [inline tensorflow/training/docker/2.7/py3/Dockerfile.ec2.cpu.os_scan_allowlist.json:29]: Can we add `reason_to_ignore` key that would make the entry look something like:

```
{
            "description": "[tensorflow](https://pypi.org/project/tensorflow/) is a machine learning framework.\n\nAffected versions of this package are vulnerable to Arbitrary File Write via Archive Extraction (Zip Slip) via a crafted archive when `tf.keras.utils.get_file` is used with `extract=True`.\r\n\r\nNOTE: This CVE is disputed as the vendor's position is that `tf.keras.utils.get_file` is not intended for untrusted archives. However, we feel this advisory is relevant as, as at the time of publication, there is no known security notice or documentation warning users of this behaviour.",
            "vulnerability_id": "IN1-PYTHON-TENSORFLOW-1315149",
            "name": "IN1-PYTHON-TENSORFLOW-1315149",
            "package_name": "tensorflow",
            "package_details": {
                "file_path": "usr/local/lib/python3.8/site-packages/tensorflow-2.7.4.dist-info/METADATA",
                "name": "tensorflow",
                "package_manager": "PYTHONPKG",
                "version": "2.7.4",
                "release": null
            },
            "remediation": {
                "recommendation": {
                    "text": "None Provided"
                }
            },
            "cvss_v3_score": 9.1,
            "cvss_v30_score": 0.0,
            "cvss_v31_score": 9.1,
            "cvss_v2_score": 0.0,
            "cvss_v3_severity": "CRITICAL",
            "source_url": "https://snyk.io/vuln/SNYK-PYTHON-TENSORFLOW-1315149",
            "source": "SNYK",
            "severity": "CRITICAL",
            "status": "ACTIVE",
            "title": "SNYK-PYTHON-TENSORFLOW-1315149 - tensorflow",
            "reason_to_ignore": "This cannot be fixed due to xyz"
```

File: tensorflow/training/docker/2.7/py3/cu112/Dockerfile.ec2.gpu.os_scan_allowlist.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            "cvss_v3_score": 7.8,
+            "cvss_v30_score": 0.0,
+            "cvss_v31_score": 7.8,
+            "cvss_v2_score": 6.8,
+            "cvss_v3_severity": "HIGH",
+            "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-1304.html",
+            "source": "UBUNTU_CVE",
+            "severity": "MEDIUM",
+            "status": "ACTIVE",
+            "title": "CVE-2022-1304 - e2fsprogs"
```
Comment (shantanutrip) [inline tensorflow/training/docker/2.7/py3/cu112/Dockerfile.ec2.gpu.os_scan_allowlist.json:29]: Same, `reason_to_ignore`

File: tensorflow/training/docker/2.7/py3/cu112/Dockerfile.ec2.gpu.os_scan_allowlist.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            "cvss_v3_score": 9.1,
+            "cvss_v30_score": 0.0,
+            "cvss_v31_score": 9.1,
+            "cvss_v2_score": 0.0,
+            "cvss_v3_severity": "CRITICAL",
+            "source_url": "https://snyk.io/vuln/SNYK-PYTHON-TENSORFLOW-1315149",
+            "source": "SNYK",
+            "severity": "CRITICAL",
+            "status": "ACTIVE",
+            "title": "SNYK-PYTHON-TENSORFLOW-1315149 - tensorflow"
```
Comment (shantanutrip) [inline tensorflow/training/docker/2.7/py3/cu112/Dockerfile.ec2.gpu.os_scan_allowlist.json:59]: Same, `reason_to_ignore`

File: tensorflow/training/docker/2.7/py3/cu112/Dockerfile.sagemaker.gpu.os_scan_allowlist.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            "cvss_v3_score": 7.8,
+            "cvss_v30_score": 0.0,
+            "cvss_v31_score": 7.8,
+            "cvss_v2_score": 6.8,
+            "cvss_v3_severity": "HIGH",
+            "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-1304.html",
+            "source": "UBUNTU_CVE",
+            "severity": "MEDIUM",
+            "status": "ACTIVE",
+            "title": "CVE-2022-1304 - e2fsprogs"
```
Comment (shantanutrip) [inline tensorflow/training/docker/2.7/py3/cu112/Dockerfile.sagemaker.gpu.os_scan_allowlist.json:29]: Same, `reason_to_ignore`

File: tensorflow/training/docker/2.7/py3/cu112/Dockerfile.sagemaker.gpu.os_scan_allowlist.json [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+            "cvss_v3_score": 9.1,
+            "cvss_v30_score": 0.0,
+            "cvss_v31_score": 9.1,
+            "cvss_v2_score": 0.0,
+            "cvss_v3_severity": "CRITICAL",
+            "source_url": "https://snyk.io/vuln/SNYK-PYTHON-TENSORFLOW-1315149",
+            "source": "SNYK",
+            "severity": "CRITICAL",
+            "status": "ACTIVE",
+            "title": "SNYK-PYTHON-TENSORFLOW-1315149 - tensorflow"
```
Comment (shantanutrip) [inline tensorflow/training/docker/2.7/py3/cu112/Dockerfile.sagemaker.gpu.os_scan_allowlist.json:59]: Same, `reason_to_ignore`

**Review by ohadkatz — APPROVED** [review]

**Review by RadhikaB-97 — APPROVED** [review]
